### DISCUSSION TITLE
Thanks for erdtree, it's a much better experience for me than tree(1)/du(1)/sort(1)/grep(1) combination hacking

### DISCUSSION THREAD

**Discussion by johnnyutahh** [solidiquis/erdtree#225] [Ideas]
[This is not an "idea" (per the Discussion category), only a "thank you note."]

Thank you @solidiquis for creating and maintaining [erdtree](https://github.com/solidiquis/erdtree). It's a much better experience for me than the tedious tree(1)/du(1)/sort(1)/grep(1) combination hacking I was previously trying through the years.

And thank you for writing it in Rust, making it easier and arguably more-reliable for any future pull requests I may submit.

**Comment by solidiquis** [comment]
Glad to hear! You just gave me some inspiration on a hook for the README so thanks for that as well! 😄

**Reply by johnnyutahh** [reply]
> You just gave me some inspiration on a hook for the README so thanks for that as well! 😄

Pleased to hear this. Also, if it helps and you're not already familiar:

Consider `fd` (also written in Rust, by a clear Rust lover: https://github.com/sharkdp), which I find to be a fantastic `find` alternative:

https://github.com/sharkdp/fd
https://stackoverflow.com/a/71931037/605356

Integrations with `fd --exec erd [...]` might be interesting and possibly wider-audience marketable. Leveraging existing or future `erd` features to attract `fd` might also be useful; it feels like there could be simliar usage paradigms between the two (`erd` and `fd`) projects.

**Reply by johnnyutahh** [reply]
...and now that I think about it: I could potentially see a world were there's some benefit in merging `erd` and `fd` into the same command. Maybe. A _very_ unstudied thought... that may or may not make sense after appropriate scrutiny.

Until then, I'm may try using `fd --exec erd [...]` a bit more.

**Reply by johnnyutahh** [reply]
Another place/project that recently added a Homebrew formula for which it might be useful to "poach" ideas (because it pitches "includes the best features of ls(1), find(1), stat(1) and du(1)", similar to my "pitches" above) that I just discovered (because it's a new Homebrew formula)... if you're not already familiar:

https://github.com/leahneukirchen/lr (C-based)

**Reply by johnnyutahh** [reply]
Additional, possibly-similar projects (Rust-based):

https://github.com/Canop/broot
https://github.com/Canop/dysk

**Reply by johnnyutahh** [reply]
Additional, similar project (Rust-based):

https://github.com/Byron/dua-cli
